pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisConsumer.java

```diff
@@ -73,11 +74,12 @@ public KinesisConsumer(KinesisConfig config, KinesisClient kinesisClient) {
    * Fetch records from the Kinesis stream between the start and end KinesisCheckpoint
    */
   @Override
-  public KinesisMessageBatch fetchMessages(StreamPartitionMsgOffset startMsgOffset, int timeoutMs) {
+  public KinesisMessageBatch fetchMessages(StreamPartitionMsgOffset startMsgOffset,
```

Comment:
Can we get #12806 in first if it works well?

Reply:
Makes sense.